conda_build/build.py

```diff
@@ -1821,8 +1821,8 @@ def _write_sh_activation_text(file_handle, m):
                             cygpath_suffix))
 
     if conda_46:
-        file_handle.write("eval \"$('{sys_python}' -m conda shell.bash hook)\"\n".format(
-            sys_python=sys.executable,
+        file_handle.write("eval \"$('{sys_python}'{I_flag} -m conda shell.bash hook)\"\n".format(
+            sys_python=sys.executable, I_flag=('' if m.config.debug else ' -I'),
         ))
```

Comment:
Ah, that's why I didn't get it to work! When I had looked at the `-m` issue previously, I thought we could simply replace `-m` with `-I`.

```suggestion
        file_handle.write(
            f"""eval "$('{sys.executable}' {'-m' if m.config.debug else '-m -I'} conda shell.bash hook)"\n"""
        )
```